packages/nextjs/src/common/utils/edgeWrapperUtils.ts

```diff
@@ -28,7 +28,7 @@ export function withEdgeWrapping<H extends EdgeRouteHandler>(
       baggage,
     });
 
-    return trace(
+    return startSpan(
```

Comment:
@lforst can you especially check that the logic for the next js spans is still correct? Just to be sure 😅

Reply:
It looks like we didn't wrap this apply call in `handleCallbackErrors`: https://github.com/getsentry/sentry-javascript/blob/75aecf6173547b82d4d57d040f6ce4f91fe4e613/packages/nextjs/src/common/wrapServerComponentWithSentry.ts#L73

Reply:
ah yeah, forgot about this - initially wrote this to just use try-catch but safer to use the util here too 👍